.github/workflows/test-install.yml

```diff
@@ -38,16 +38,13 @@ jobs:
         python-version: '3.11'
 
     - name: Set up uv
-      uses: astral-sh/setup-uv@v5.2.0
+      uses: astral-sh/setup-uv@v5.2.1
       with:
-        version: 0.5.x
+        version: latest
```

Comment:
why do we use here latest and not version 0.5.22?

Reply:
Please see: https://github.com/aiidateam/aiida-core/pull/6699#discussion_r1924592779

We're only using the `uv pip install` interface here so we can keep things simple.

Reply:
> Also in that action python-version: ${{ inputs.python-version }} is specified. Should we add here also a python-version: 3.11 ?

In the actions case, specifying the python version is needed because it has a side-effect of automatically activating the virtual environment. Again, this is not needed here we can keep things simple and not duplicate the python version information.

Reply:
I see thanks for the clarification